src/stickybits.js

```diff
@@ -71,7 +71,7 @@ function Stickybits(target, obj) {
     -  uses a computed (`.definePosition()`)
     -  defined the position
   */
-  p.positionVal = this.definePosition() || 'fixed'
```

Comment:
`.definePosition()` defines the position. Defining the position (whether it is `sticky` or `fixed`) should happen within the `.definePosition()`.

Reply:
So this change should be included in the new PR?